### DISCUSSION TITLE
Trying to get verified as a student  but can't upload documents, only take janky photos with the selfie camera on my phone.

### DISCUSSION THREAD

**Discussion by randyb100** [community/community#52070] [GitHub Education]
I'm a university student online. I go to TESU in NJ for a bachelor program in computer science. I tried to get approved last night, and their application page has this janky camera feature that doesn't work well. It doesn't work at all on my laptop, and when I tried on my phone, it only allows me to use the front facing camera! I have to point and click blind, and photos end up at weird angles. Why can't I just upload documents?

Is there an email address I can send documents to? I can get a National Clearinghouse verification of enrollment letter. It seems it's become difficult to get approved lately. Any tips?

**Comment by samus-aran** [comment]
Hi @randyb100  👋

We're happy you're here!

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance

**Reply by randyb100** [reply]
It's not having access to high quality cameras, it's using the website to take photos. It JUST lets me use the front facing camera. I tried submitting multiple documents and they all failed. I submitted a letter from the National Student Clearinghouse that verifies that I'm a 3/4 time student at TESU. I submitted a financial statement from the bursar's office, and a screenshot of the student portal showing I'm enrolled in 3 classes. None of them worked.

**Reply by samus-aran** [reply]
Hi @randyb100 

Thanks for that feedback, we'll pass that onto the internal team regarding the front facing camera. 

May I offer you additional advice on your next application submission. Its a common reflection based on your public profile name I can see. 

The application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process. Is your name Randy or is this a nickname? Consider that when you are comparing your GitHub profile details and your evidence.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Another option is to take photo of multiple pieces of evidence e.g. a Student ID on top of or alongside a transcript.

Let me know how you get on 👍

**Reply by Saran-pariyar** [reply]
Hi! 
I tried multiple times but it's not getting clear image of the ID card. I clicked the image of my ID card but it's not accepted and it's said to take photo using browser camera but it doesn't takes clear image. What should I do?

**Reply by JeffSwe** [reply]
I'm in the same boat, due to the website not using the camera app the quality is horrendous and I can't zoom, neither can I focus the camera.
Its as if you're poor then the message is fuck you I guess.

**Reply by khrnchn** [reply]
please help, its only a few months left before i can enjoy the perks of github's student pack. i tried using both my phone and my pc's camera to snap my doc/ id card but cant. the cameras are inverted. when i want to use the back camera option when using phone, nothing shows up. please fix it/ allow us to upload via back camera.

**Reply by VadoVates** [reply]
Meh, I have the same problem and nothing worked. I've just gave up. Sorry mate but to be honest, propably GitHub isn't interested in solving the problem.

**Reply by Bedrock16** [reply]
Absolutely true the issue is since a year now I think and that's not a very hard thing to solve but they don't bother to look into it.

**Reply by VadoVates** [reply]
Last week I took documents from my university, which have told I'm a student now. All the data in official paper, signed by university. Same as before, discarded as not enough proof.
Polish student ID is also totally useless, because on one side it has data with student info, on the other side it has (official state hologram) stickers with validity. That's too much for GitHub, because you don't have it all on one picture.
